backend/src/main/java/com/ideahub/model/IdeaPartType.java

```diff
@@ -42,7 +42,7 @@
     @JsonIgnore
     private int id;
     
-    @Column(name = "name", nullable = false)    
+    @Column(name = "name", nullable = false) //Is this going to be a unique value?
```

Comment:
Yes this will be unique. It's purely for us to reference so we don't need to remember ids.

The whole table is only ever touched by us.